package.json

```diff
@@ -31,7 +31,7 @@
   "author": "Microsoft Conversation Learner Team",
   "license": "MIT",
   "dependencies": {
-    "@conversationlearner/sdk": "0.315.2",
```

Comment:
What downstream package change was this dependent on?

From the title / description I thought this was only a re-export and save of the model generated by the tutorial?

Also, the PR is targeting `master` instead of `develop`. It was my understanding this change was not critical enough to require push out to `master` and release

Reply:
To be clear I'm expecting the `package.json` and `package.lock.json` to be unchanged

Reply:
The package* files should not have been included in the PR. Deleting this branch as it's a bit stale and reworking with single sample .json file change